packages/netlify-cms-core/src/components/Editor/EditorInterface.js

```diff
@@ -221,7 +221,14 @@ class EditorInterface extends Component {
     } = this.props;
 
     const { scrollSyncEnabled, showEventBlocker } = this.state;
-    const collectionPreviewEnabled = collection.getIn(['editor', 'preview'], true);
+    let previewEnabled;
+    if (collection.get('type') === 'file_based_collection') {
```

Comment:
You can use the `const` value from:
https://github.com/netlify/netlify-cms/blob/2146b3605f337e808f0e54c92cd2b601296f23a6/packages/netlify-cms-core/src/constants/collectionTypes.js#L1